react/components/molecules/key-values/key-values.js

```diff
@@ -156,6 +156,36 @@ export class KeyValues extends PureComponent {
                                 {item.valueComponent}
                             </KeyValue>
                         )}
+                        {item.section && (
+                            <View>
+                                <Text style={styles.sectionTitle}>{item.key}</Text>

```

Comment:
this seems to be duplicate just to show the section... can't we just add that condition for <Text> like:
```suggestion
                                {item.section && <Text style={styles.sectionTitle}>{item.key}</Text>}
```

and above if you want a different style make that condition inside the style?
this would avoid duplicating this code